.github/workflows/ci.yml

```
@@ -0,0 +1,70 @@
+name: cppfront CI
```

[Suggestion] I think the general practice with GitHub workflows is to make both the YAML file name and the name displayed in the GitHub UI more descriptive. How about something like `build-cppfront.yaml` and:
```suggestion
name: Multi-platform Build of cppfront
```
or something like that?

---

Done!